uwblueprint/ccbc
# [bug] Remove Shadow Chasers from Create Review page

## Notion ticket link

[Remove Shadow Chasers](https://www.notion.so/uwblueprintexecs/Remove-Shadow-Chasers-0ea75e1504db47bd98650684ba265e4d)



## Implementation description
* Removed the Shadow Chasers book from the create review page



## Steps to test
1. create a review, check that shadow chasers is no longer there
2. check that you can still successfully publish a review



## What should reviewers focus on?
* check that shadow chasers is not there


## Checklist
